docs/source/variables.adoc

```diff
@@ -1,2 +1,2 @@
-:minishift-version: unset
-:openshift-version: unset
+:minishift-version: 1.0.0-rc.1
```

Comment:
I didn't touch the variables.adoc file so maybe this is something from the rebase merge? not sure how to fix this.